runUItests.sh

```diff
@@ -22,6 +22,6 @@ npm run compile-tests
 # test what was compiled
 
 exec npx extest setup-and-run out/test/vscode-suite/*.test.js \
-    --code_version 1.82.3 \
+    --code_version 1.98.0 \
```

Comment:
why are we going this high? we want to make sure we're running against the "oldest" version we can support.

Reply:
The `runUITests.sh` script now takes the version number from `package.json`.